applications/AdjointFluidApplication/tests/test_vms_sensitivity_2d/cylinder_test_adjoint_parameters.json

```diff
@@ -139,7 +139,8 @@
             "position"         : [0.020957, 0.0055272, 0.0],
             "model_part_name"  : "Parts_Fluid",
             "output_file_name" : "test_vms_sensitivity_2d/cylinder_test_adjoint_probe1.dat",
-            "output_variables" : ["ADJOINT_VELOCITY_X", "ADJOINT_VELOCITY_Y"]
+            "output_variables" : ["ADJOINT_VELOCITY_X", "ADJOINT_VELOCITY_Y"],
+            "save_output_in_folder"  : false
```

Comment:
i have the impression that this belongs in a different PR...

Reply:
Will move to a different PR